### PR TITLE
Index items page

### LINKED ISSUE #17: User Story 17, Items Index Page
As any kind of user on the system
I can visit the items catalog ("/items")
I see all items in the system except disabled items

The item image is a link to that item's show page

### PR DESCRIPTION
**Pull request by stillsheryl**
This PR will:
- Make only items that are active show on the items index page
- Make the image a clickable link to the item index page
- All tests passing

- closes #17

### CODE REVIEW

**Review by c-bartell — APPROVED** [review]

**Review by JoseLopez235 — APPROVED** [review]
LGTM

File: spec/features/items/index_spec.rb [hunk 1]
```diff
@@ -19,11 +19,9 @@
       expect(page).to have_link(@tire.merchant.name)
       expect(page).to have_link(@pull_toy.name)
       expect(page).to have_link(@pull_toy.merchant.name)
-      expect(page).to have_link(@dog_bone.name)
-      expect(page).to have_link(@dog_bone.merchant.name)
     end
 
-    it "I can see a list of all of the items "do
+    it "I can see a list of all of the active items, while the inactive items are not listed"do
```
Comment (KieraAllen) [inline spec/features/items/index_spec.rb:24]: So this is for the general items index? Makes sense to me!

File: spec/features/items/index_spec.rb [hunk 2]
```diff
@@ -47,15 +45,23 @@
...
+      expect(page).to_not have_content(@dog_bone.description)
+      expect(page).to_not have_content("Price: $#{@dog_bone.price}")
+      expect(page).to_not have_content("Inactive")
+      expect(page).to_not have_content("Inventory: #{@dog_bone.inventory}")
+    end
+
+    it "The item image is a link to that item's show page" do
+      visit '/items'
+
+      find("#link-#{@tire.id}").click
```
Comment (KieraAllen) [inline spec/features/items/index_spec.rb:58]: SO glad I know how to do this now! Thank you 😄

File: spec/features/items/merchant_items_index_spec.rb [hunk 3]
```diff
@@ -30,14 +30,12 @@
         expect(page).to have_content("Inventory: #{@chain.inventory}")
       end
 
-      within "#item-#{@shifter.id}" do
```
Comment (KieraAllen) [inline spec/features/items/merchant_items_index_spec.rb:33]: Was this code given to us at the start of the project?

**Review by KieraAllen — APPROVED** [review]

**Pull request merged**